api/openapi.yaml

```diff
@@ -457,6 +457,74 @@ paths:
           $ref: "#/components/responses/401"
         404:
           $ref: "#/components/responses/404"
+  /users/{userId}/tags:
+    post:
+      operationId: addTag
+      tags:
+        - "Tags"
+      description: A user can add tag to his own profile.
+      summary: "Add tag to user profile"
+      parameters:
+        - in: path
+          name: userId
+          schema:
+            type: integer
+            format: int64
+          required: true
+          description: Numeric ID of the user
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Tag'
+            encoding:
+              tagDto:
+                contentType: application/json
+        description: "Tag object that needs to be created"
+      responses:
+        201:
+          description: The tag was succesfully created
+        400:
+          $ref: "#/components/responses/400"
+        401:
+          $ref: "#/components/responses/401"
+        403:
+          $ref: "#/components/responses/403"
```

Comment:
Is there no `404` response here, e.g. when the userId does not exist on the server?

Reply:
Added in commit def59ab, thanks!